### PR TITLE
[DOCS-6131] Add Ansible deployment

### PR DESCRIPTION
**Pull request by anxumalo**
TODO: Fix link to new securing install page (from #164 )

### CODE REVIEW

File: content-services/latest/install/ansible.md [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+
+Before continuing we need to introduce some [Ansible concepts](https://docs.ansible.com/ansible/latest/user_guide/basic_concepts.html); [control node](https://docs.ansible.com/ansible/latest/user_guide/basic_concepts.html#control-node), [connection type](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#connecting-to-hosts-behavioral-inventory-parameters) and the [inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#intro-inventory).
+
+The machine the playbook is run from is known as the **control node**. An **inventory** file is used to describe where the different Content Services should be installed on. 
+
+There are two types of installation: 
+* **Local** where all the components will be installed on the control node machine:
+
+![](../../images/deployment-type-local.png)
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:16]: Image is not visible and I think the image link is not specified correctly, should be something like:
    deployment-type-local

File: content-services/latest/install/ansible.md [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+
+* **Remote** (also known as `ssh`) where components can be installed on one or more remote `hosts`. These hosts can be bare metal machines, Virtual machines or instances running on a public cloud:
+
+![](../../images/deployment-type-ssh.png)
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:20]: Image is not visible and I think the image link is not specified correctly, should be something like:
    deployment-type-ssh

File: content-services/latest/install/ansible.md [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+
+# Setting up Ansible
+
+As mentioned in the introduction a control node is required to run the playbook. You can use any computer that has a Python installation as a control node; laptops, shared desktops, and servers can all run Ansible.
+
+In the interest of keeping this guide simple we will use an AWS EC2 instance as the control node, the steps required are shown below:
+
+1. Launch an EC2 instance using the Centos 7 or 8 (x86_64) AMI from the Marketplace (instance size/type does not matter)
+
+    ![](../../images/centos-ami.png)
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:36]: Image is not visible and I think the image link is not specified correctly, should be something like:
centos-ami

File: content-services/latest/install/ansible.md [hunk 4]
```diff
@@ -0,0 +1,489 @@
...
+Without any additional configuration applied, the playbook will install the default Alfresco Content Services components. Please review the [configuration](#configure-your-installation) section below, to adjust some of the configurable installation options.
+
+To install everything on the control node follow the steps in the [Locahost Installation](#Local-Installation) section or to install to one or more other machines follow the steps in the [Remote Installation](#Remote-Installation) section.
+
+
+# Local Installation
+
+The diagram below shows the result of a local installation.
+
+![](../../images/acs-localhost.png)
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:81]: Image is not visible and I think the image link is not specified correctly, should be something like:
acs-localhost

File: content-services/latest/install/ansible.md [hunk 5]
```diff
@@ -0,0 +1,489 @@
...
+      -o ControlPersist=60s -o ForwardX11=no -o LogLevel=ERROR
+      -o IdentitiesOnly=yes -o StrictHostKeyChecking=no"
+```
+
+
+## Single Machine Installation
+
+The diagram below shows the result of a single machine installation.
+
+![](../../images/acs-single-machine.png)
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:144]: Image is not visible and I think the image link is not specified correctly, should be something like:
acs-single-machine

File: content-services/latest/install/ansible.md [hunk 6]
```diff
@@ -0,0 +1,489 @@
...
+* Digital Workspace: `http://<target-host-ip>/workspace`
+* Share: `http://<target-host-ip>/share`
+* Repository: `http://<target-host-ip>/alfresco`
+* API Explorer: `http://<target-host-ip>/api-explorer`
+
+## Multi Machine Installation
+
+The diagram below shows the result of a multi machine installation.
+
+![](../../images/acs-multi-machine.png)
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:193]: Image is not visible and I think the image link is not specified correctly, should be something like:
acs-multi-machine

File: content-services/latest/install/ansible.md [hunk 7]
```diff
@@ -0,0 +1,489 @@
...
+7. Create environment variables to hold your Nexus credentials as shown below (replacing the values appropriately):
+
+    ```bash
+    export NEXUS_USERNAME="<your-username>"
+    export NEXUS_PASSWORD="<your-password>"
+    ```
+
+Without any additional configuration applied, the playbook will install the default Alfresco Content Services components. Please review the [configuration](#configure-your-installation) section below, to adjust some of the configurable installation options.
+
+To install everything on the control node follow the steps in the [Locahost Installation](#Local-Installation) section or to install to one or more other machines follow the steps in the [Remote Installation](#Remote-Installation) section.
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:74]: page links #Local-Installation and #Remote-Installation does not work, needs to be #local-installation an #remote-installation

File: content-services/latest/install/ansible.md [hunk 8]
```diff
@@ -0,0 +1,489 @@
...
+
+Along with the url the database driver binaries need to be provided for one or both services in the `configuration_files/db_connector_repo` and/or `configuration_files/db_connector_sync` folders.
+
+The default database username (`repo_db_username` and/or `sync_db_username`) and password (`repo_db_password` and/or `sync_db_password`) in the configuration file `group_vars/all.yml` can also be overidden with your custom values.
+
+Please refer to the [Configuring Databases](https://docs.alfresco.com/6.2/concepts/intro-db-setup.html) documentation for more detailed information.
+
+## Custom Keystore
+
+By default the playbook installs a default keystore to ease the installation process, however, we recommend you [generate your own keystore](https://docs.alfresco.com/6.2/concepts/keystore-generate.html) following the [instructions here](https://docs.alfresco.com/6.2/concepts/keystore-config.html).
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:363]: Incorrect links and format: https://docs.alfresco.com/6.2/concepts/keystore-generate.html and https://docs.alfresco.com/6.2/concepts/keystore-config.html

File: content-services/latest/install/ansible.md [hunk 9]
```diff
@@ -0,0 +1,489 @@
...
+```yaml
+repo_db_url: jdbc:mysql://54.164.117.56:3306/alfresco?useUnicode=yes&characterEncoding=UTF-8
+repo_db_driver: com.mysql.jdbc.Driver
+```
+
+Along with the url the database driver binaries need to be provided for one or both services in the `configuration_files/db_connector_repo` and/or `configuration_files/db_connector_sync` folders.
+
+The default database username (`repo_db_username` and/or `sync_db_username`) and password (`repo_db_password` and/or `sync_db_password`) in the configuration file `group_vars/all.yml` can also be overidden with your custom values.
+
+Please refer to the [Configuring Databases](https://docs.alfresco.com/6.2/concepts/intro-db-setup.html) documentation for more detailed information.
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:359]: Incorrect link and format: https://docs.alfresco.com/6.2/concepts/intro-db-setup.html

File: content-services/latest/install/ansible.md [hunk 10]
```diff
@@ -0,0 +1,489 @@
...
+
+By default, without any configuration applied, the playbook will install a limited trial of the Enterprise version of Alfresco Content Services 7.x that goes into read-only mode after 2 days. If you'd like to try Alfresco Content Services for a longer period, request the 30-day [Download Trial](https://www.alfresco.com/platform/content-services-ecm/trial/download).
+
+The sections below describe how you can configure your installation before running the playbook.
+
+## License
+
+If you have a valid license place your `*.lic` file in the `configuration_files/licenses` folder before running the playbook.
+
+>NOTE: You can also [upload a license](https://docs.alfresco.com/6.2/tasks/at-adminconsole-license.html) via the Admin Console once the system is running.
```
Comment (gravitonian) [inline content-services/latest/install/ansible.md:296]: Incorrect link and format: https://docs.alfresco.com/6.2/tasks/at-adminconsole-license.html

**Review by gravitonian — CHANGES_REQUESTED** [review]

File: content-services/community/install/ansible.md [hunk 11]
```diff
@@ -0,0 +1,390 @@
...
+| `/var/log/alfresco` | Logs |
+
+### Service configuration
+
+The following `systemd` services are installed, which you can use to stop and start each service:
+
+| Service Name | Purpose |
+| ------------ | ------- |
+| `activemq.service` | ActiveMQ Service |
+| `postgresql-<version>.service` | PostgreSQL DB Service (where `<version>` is 11 for ACS 6.2.N and 13 for ACS 7.x) |
```
Comment (anxumalo) [inline content-services/community/install/ansible.md:221]: @gavincornwell In this Community page, should we simplify this line to just `postgres-13.service` (to match line 364?

Comment (gavincornwell) [reply content-services/community/install/ansible.md:221]: Yes please.

**Pull request merged**